figures/S7.supplemental_figure7/scripts/S7.r

```diff
@@ -0,0 +1,288 @@
+    replacement = "pool",
+    x = metadata_file_SHSY5Y$label
+)
+metadata_file_SHSY5Y$label <- gsub(
+    pattern = "train",
+    replacement = "pool",
+    x = metadata_file_SHSY5Y$label
+)
+metadata_file_SHSY5Y$label <- gsub(
+    pattern = "val",
+    replacement = "pool",
+    x = metadata_file_SHSY5Y$label
+)
+# aggreagate the metadata via Metadata_Well
+metadata_file_SHSY5Y <- metadata_file_SHSY5Y %>%
+    group_by(Metadata_Well, Metadata_cell_type, label) %>%
+    summarize(
+        n = n()
+    )
+head(metadata_file_SHSY5Y)
+
+
+# combine the SHSY5Y and PBMC metadata files
+combined_metadata <- rbind(
+    metadata_file_PBMC,
+    metadata_file_SHSY5Y
+)
+head(combined_metadata)
+unique(combined_metadata$Metadata_cell_type)
+unique(combined_metadata$label)
+
+
+
+
+
+# load in the platemap
+platemap_file_path <- file.path(
+    "../",
+    "../",
+    "../",
+    "data/",
+    "Interstellar_plate2_platemap.csv"
+)
+# read in the platemap
+platemap_file <- read.csv(
+    file = platemap_file_path,
+    header = TRUE,
+    stringsAsFactors = FALSE
+)
+head(platemap_file)
+
+# merge the combined metadata file with plate map on Metadata_Well
+updated_platemap <- merge(
+    x = platemap_file,
+    y = combined_metadata,
+    by.x = "well_id",
+    by.y = "Metadata_Well",
+    all.x = TRUE
+)
+head(updated_platemap)
+unique(updated_platemap$Metadata_cell_type)
+# replace "" in cell_type with "Blank" in label
+# updated_platemap$Metadata_cell_type[updated_platemap$Metadata_cell_type == NA] <- "Blank"
+# replace NA in cell_type with "Blank" in label
+updated_platemap$Metadata_cell_type[is.na(updated_platemap$Metadata_cell_type)] <- "Blank"
+# replace NA with "Blank" in label
+updated_platemap$label[is.na(updated_platemap$label)] <- "Blank"
+# replace pool with Pool in label
+updated_platemap$label[updated_platemap$label == "pool"] <- "Pool"
+# replace holdout with Holdout in label
+updated_platemap$label[updated_platemap$label == "holdout"] <- "Holdout"
+# replace treatment_holdout with Treatment Holdout in label
+updated_platemap$label[updated_platemap$label == "treatment_holdout"] <- "Treatment Holdout"
+unique(updated_platemap$Metadata_cell_type)
+unique(updated_platemap$label)
+
+# Make the cell type factor ordered
+updated_platemap$Metadata_cell_type <- factor(
+    x = updated_platemap$Metadata_cell_type,
+    levels = c(
+        "Blank",
+        "PBMC",
+        "SH-SY5Y"
+    )
+)
+# make the label factor ordered
+updated_platemap$label <- factor(
+    x = updated_platemap$label,
+    levels = c(
+        'Blank',
+        'Pool',
+        'Holdout',
+        'Treatment Holdout'
+    )
+)
+
+width <- 14
+height <- 14
+options(repr.plot.width = width, repr.plot.height = height, units = "cm")
+# set pallette
+viridis_pal_custom <- viridis::viridis_pal(option = "C")(3)
+
+data_split_plate_map <- (
+    raw_map(
+        data = updated_platemap$label,
+        well = updated_platemap$well_id,
+        plate = 384,
+        size = 14)
```

Comment:
Consider documenting why this plate and size number (if that hasn't already been done elsewhere).

Reply:
addressed